cloudamqp/resource_cloudamqp_extra_disk_size.go

```diff
@@ -28,27 +26,103 @@ func resourceExtraDiskSize() *schema.Resource {
 				Required:    true,
 				Description: "Extra disk size in GB",
 			},
+			"allow_downtime": {
+				Type:        schema.TypeBool,
+				ForceNew:    true,
+				Required:    true,
+				Description: "When resizing disk, allow downtime to do so",
```

Comment:
Should we be more specific about what kind of downtime? Is it the whole cluster or only one node at a time?

Like

```suggestion
				Description: "When resizing disk, allow cluster downtime to do so",
```

Or 

```suggestion
				Description: "When resizing disk, allow node downtime to do so",
```

Reply:
Single node cluster will go down. Multi node cluster will close connections on the node being the leader when the disk is swapped and pass on the leadership to one of the others. This happens until all nodes have gotten their disk swapped. So single node (cluster down) and multi node (node down).